### DISCUSSION TITLE
Recent and Hot topics

### DISCUSSION THREAD

**Discussion by demonlibra** [blitze/phpBB-ext-sitemaker#68] [General]
Good day.
What is the difference between Recent and Hot topics?

**Comment by blitze** [comment]
Hi demonlibra,

That is a good question. phpBB has this concept of "Hot topics". These are topics that have at least X number of replies, where X is defined in ACP > Post Settings > Popular topic threshold.  You recognize them because they have the below icon:

![image](https://user-images.githubusercontent.com/21600/110135394-17088c00-7d94-11eb-993f-f528f2a3d301.png)

I debated whether to call this "Hot Topics" or "Popular Topics". From your question, it looks like "Popular Topics" might be a better name for it. What do you think?

Thanks

**Reply by demonlibra** [reply]
Maybe the answer will be different in different situations and different forums. Everyone can change the title (or language file) as they like.
Thank you.
